### PR TITLE
Issue 462 - Broker::parseEventForOrganizer copies DTSTAMP from $eventInfo that causes broken scheduling

### PR DESCRIPTION
**Pull request by alexander-ivannikov**
Implementation of a fix for https://github.com/sabre-io/vobject/issues/462

### CODE REVIEW

**Comment by staabm** [comment]
It seems this requires some changes to test-expectations.

**Comment by staabm** [comment]
you are on the right track. just one unit test failure left :-)

**Comment by alexander-ivannikov** [comment]
> you are on the right track. just one unit test failure left :-)

Looks like I have finally fixed all of the failing unit tests. But not the entire CI build...

File: tests/VObject/ITip/EvolutionTest.php [hunk 1]
```diff
@@ -1719,6 +1717,7 @@ public function testNewEvolutionEvent()
  TRUE;LANGUAGE=en:MAILTO:dominik@fruux.com
 CREATED:20140813T153211Z
 LAST-MODIFIED:20140813T155353Z
+DTSTAMP:**ANY**
```
Comment (vimmerru) [inline tests/VObject/ITip/EvolutionTest.php]: @alexander-ivannikov We can check that DTSTAMP was increased and it is more than moment before parseEventForOrganizer call

**Comment by staabm** [comment]
thx - we will do the phpstan in a separate PR.

**Pull request merged**